doc/06-distributed-monitoring.md

```diff
@@ -3230,6 +3230,47 @@ information/pki: Writing certificate to file 'icinga2-satellite1.localdomain.crt
 
 Copy and move these certificates to the respective instances e.g. with SSH/SCP.
 
+#### External CA/PKI
+
+Icinga works best with its own certificates.
+The commands described above take care of the optimal certificate properties.
+Also, Icinga renews them periodically at runtime to avoid expiry.
+But you can also provide your own certificates,
+just like to any other application which uses TLS.
+
+!!! warning
+
+    The only serious reasons to generate own certificates are company policies.
+    You are responsible for making Icinga working with your certificates,
+    as well as for [expiry monitoring](10-icinga-template-library.md#plugin-check-command-ssl_cert)
+    and renewal. Especially the CLI doesn't expect such certificates.
+    Apropos renewal, don't provide the CA private key in `/var/lib/icinga2/ca`!
+    Icinga uses it to automatically renew certificates with standard properties,
+    not any custom ones.
```

Comment:
I would split this paragraph as it contains lots of important information about different topics.

```suggestion
    The only serious reasons to generate own certificates are company policies.
    You are responsible for making Icinga working with your certificates,
    as well as for [expiry monitoring](10-icinga-template-library.md#plugin-check-command-ssl_cert)
    and renewal.
    
    Especially the CLI doesn't expect such certificates.
    
    Apropos renewal, don't provide the CA private key in `/var/lib/icinga2/ca`!
    Icinga uses it to automatically renew certificates with standard properties,
    not any custom ones.
```

Then, please clarify what you are meaning when talking about CLI tools. And maybe state that the CA should be within `/var/lib/icinga2/certs/ca.crt` (as mentioned later on) and not only where it shouldn't be.